robot/Cumulus/resources/locators_49.py

```diff
@@ -241,29 +241,37 @@
         'select_related':'//select[@id = "DomainEnumOrId"]',
         'select_related_option':'//select[@id = "DomainEnumOrId"]/option[@value="{}"]',
         'search_result': '//tbody/tr/td/a/span[contains(text(),"{}")]',
+        'formula_txtarea': '//textarea[@id = "{}"]',
+        'input_validation': '//input[@title = "Check Syntax"]',
```

Comment:
please pass 'Check Syntax' as variable and use locator in line 239

Reply:
done